build/yamls/antrea-node-init.yml

```diff
@@ -0,0 +1,42 @@
+kind: DaemonSet
+apiVersion: extensions/v1beta1
```

Comment:
DaemonSet in the extensions/v1beta1 and apps/v1beta2 API versions is no longer served since 1.16.
DaemonSet in the apps/v1 API version is available since v1.9, I think you should this one?

Reply:
Done